Batteries/Data/Vector/Basic.lean

```diff
@@ -0,0 +1,447 @@
+import Batteries.Data.Array
+import Batteries.Data.List.Basic
+import Batteries.Data.List.Lemmas
+
+namespace Vector
+
+open Array Lean
+
+/--
+  `Vector α n` is an `Array α` whose size is statically fixed to `n`
+-/
+structure Vector (α : Type u) (n : Nat) where
```

Comment:
Should be outside the `Vector` namespace.

Reply:
Done